src/routes/safe/components/Apps/types.ts

```diff
@@ -11,3 +11,11 @@ export type SafeApp = Omit<SafeAppData, 'id'> & {
 export type StoredSafeApp = {
   url: string
 }
+
+export type AppTrackData = {
+  string: {
+    timestamp: Date
+    openCount: number
+    txCount: number
+  }
+}
```

Comment:
I would be more explicit with types like this:

```suggestion
export type AppTrackData = {
  [safeAppId: string]: {
    timestamp: Date
    openCount: number
    txCount: number
  }
}
```